src/sections/featured-product.liquid

```diff
@@ -2,54 +2,91 @@
 {%- assign current_variant = product.selected_or_first_available_variant -%}
 {%- assign featured_image = current_variant.featured_image | default: product.featured_image -%}
 
-{% if product == empty %}
-  {%- assign section_onboarding = true -%}
-  {%- assign onboarding_title = 'homepage.onboarding.product_title' | t -%}
-{% endif %}
+{%- if product == empty -%}
+  {%- assign default_product_title = 'homepage.onboarding.product_title' | t -%}
+{%- endif -%}
+
+<section data-section-id="{{ section.id }}" data-section-type="product" class="featured-product">
+
+  <div class="featured-product__image-container">
+    {%- unless product.images == empty -%}
+      {%- for image in product.images -%}
+        {%- capture wrapper_class -%}
```

Comment:
Do we need a capture here? I could understand if we have multiple values and conditions, but the contents of this are pretty simple and could fit as an `{% assign %}`